perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -118,7 +114,32 @@
 
 
 FLAGS_IGNORED_FOR_CUSTOM_JOBFILE = {
-    'generate_scenarios', 'io_depths', 'run_for_minutes'}
+    'fio_generate_scenarios', 'fio_io_depths', 'fio_run_for_minutes'}
+
+
+# Modes from --fio_target_mode
+AGAINST_DEVICE_MODES = {'against_device_with_fill',
+                        'against_device_without_fill'}
+FILL_TARGET_MODES = {'against_device_with_fill', 'against_file_with_fill'}
+AGAINST_DEVICE_WITH_FILL_MODE = 'against_device_with_fill'
```

Comment:
Please define this constant at the top of this function, and then use this constant in all the places where "against_device_with_fill" string is referenced, e.g., in the AGAINST_DEVICE_MODES array.

Also, define constants for all similar strings like: "against_device_without_fill", "against_file_with_fill", etc. This will provide a pretty effective guard against miss spelling of these strings. In all places where these strings are referenced in this code, they should be changed to refer to the global variables instead.